code/modules/games/cards.dm

```diff
@@ -440,7 +464,11 @@
 	var/datum/playingcard/card = pickablecards[pickedcard]
 	if(loc != user) // Don't want people teleporting cards
 		return
-	user.visible_message("<span class='notice'>[user] draws a card from [user.p_their()] hand.</span>", "<span class='notice'>You take the [pickedcard] from your hand.</span>")
+	user.visible_message(
+		"<span class='notice'>[user] draws a card from [user.p_their()] hand.</span>",
+		"<span class='notice'>You take the [pickedcard] from your hand.</span>",
```

Comment:
Grabbing the name directly instead of doing `[src]` will not generate a "the". I *have* made it \the due to custom cards that may be proper nouns (this will come up in the blank cards in our own-brand Cards Against Humanity).